docs/testing-guidelines/getting-code-coverage.md

```diff
@@ -0,0 +1,129 @@
+# Getting Code Coverage Analysis for PowerShell
+
+**Note: Code coverage is currently only supported on Windows, since we use OpenCover.**
+
+The PowerShell code base is configured to build with code coverage support using [OpenCover].
+
+You can see the testing coverage of the current [`master`] branch build at any time at [codecov.io].
+
+To run test coverage analysis of PowerShell on your own branch/machine,
+you will need to take the following steps
+(and be aware that running the code coverage analysis can take as long as 8 hours).
+
+## Running tests with code coverage analysis
+
+**First**: Open PowerShell in an **elevated** session.
+OpenCover needs elevated privileges to work.
+
+Now, in PowerShell:
+
+```powershell
+# Go to your PowerShell build directory root
+PS> Set-Location "C:\Path\to\powershell\build\dir"
+
+# Import the PowerShell build module
+PS> Import-Module .\build.psm1
+
+# Build PowerShell. You may need to add other flags here like
+# -ResGen, -Restore or -PSModuleRestore
+PS> Start-PSBuild -Configuration CodeCoverage
```

Comment:
I would add `-PSModuleRestore` in the example instead of comment, since it will be required for some tests. Maybe even `-Clean`.